data/init.sql

```diff
@@ -192,3 +191,21 @@ CREATE TABLE IF NOT EXISTS announcements (
 	created_at TIMESTAMP,
 	updated_at TIMESTAMP
 );
+
+CREATE TABLE IF NOT EXISTS teams (
+	id INT(11) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+	lecture_id INT(11) UNSIGNED,
+	FOREIGN KEY(lecture_id) REFERENCES lectures(id) ON DELETE CASCADE,
+	created_at TIMESTAMP,
+	updated_at TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS team_members (
+	team_id INT(11) UNSIGNED,
+	user_id INT(11) UNSIGNED,
+	FOREIGN KEY (user_id) REFERENCES users(id) ON DELETE CASCADE,
+	FOREIGN KEY(team_id) REFERENCES teams(id) ON DELETE CASCADE,
+	PRIMARY KEY (team_id, user_id),
+	created_at TIMESTAMP,
+	updated_at TIMESTAMP
+);
```

Comment:
the sql script doesn't really work, gotta fix it

Reply:
fixed